readthedocs/templates/core/project_details.html

```diff
@@ -46,9 +46,11 @@ <h3>{% trans "Versions" %}</h3>
         </li>
 
       {% empty %}
-      <p>
-      {% trans "No active versions." %}
-      </p>
+        <li class="module-item">
+          <p class="quiet">
+            {% trans "No active versions." %}
```

Comment:
Why are we considering this as a list of 1 element if it's empty?

I mean, we are using a `li` and displaying it as a row when it's not a list. In another comment I suggested to use just the simple text:

"No active versions."

as a `p` without too much style.

Reply:
This is a common pattern across other lists (some times with a `p` and other just with a `li`)

https://github.com/rtfd/readthedocs.org/blob/499a74076f0debae09acc1d4c4eeeab03f8f557e/readthedocs/templates/core/project_list_featured.html#L32

https://github.com/rtfd/readthedocs.org/blob/2f6b36d0a5f38ea3cf5d8998520ac095798f9b1c/readthedocs/templates/projects/integration_list.html#L37

https://github.com/rtfd/readthedocs.org/blob/d7abdd6352773eee96f6d02fca4c1c9757539ac6/readthedocs/templates/core/filter_list.html#L12

Reply:
Nice reply! Hehe!

Makes sense to leave as you did, then :)